packages/fxa-admin-server/src/app.module.ts

```diff
@@ -80,10 +79,13 @@ const version = getVersionInfo(__dirname);
       provide: APP_GUARD,
       useClass: UserGroupGuard,
     },
-    {
-      provide: LOGGER_PROVIDER,
-      useClass: MozLoggerService,
-    },
+    // {
```

Comment:
Does this need to be commented out or can be removed?

Reply:
I'll remove it.